TERRAFORM CRASH with Terraform v0.9.10 running terraform graph

I'm a n00b terraform user.  Tried terraform graph for the first time.  "terraform graph" worked (didn't crash).  "terraform graph -type=apply /tmp" caused the attached crash.

### Panic Output
https://gist.github.com/miketruty/0f3522609d05c3d687f5abdba2526d76

### Expected Behavior
Not sure.  Using graph for the first time. I was hoping for a pretty picture.

### Actual Behavior
panic: runtime error: invalid memory address or nil pointer dereference

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. terraform graph -type=apply /tmp

### Important Factoids
No

### References
No

### Call Stack
[terraformconfig.zip](https://github.com/hashicorp/terraform/files/1151425/terraformconfig.zip)

truty@gcp-automated-networks:~/gcp-automated-networks/terraform$ t graph -type=apply /tmp
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x9e1f6d]

goroutine 1 [running]:
github.com/hashicorp/terraform/terraform.(*OrphanOutputTransformer).transform(0xc420320470, 0xc4202cfcc0, 0x0, 0x0, 0xc4202cc340)
        /opt/gopath/src/github.com/hashicorp/terraform/terraform/transform_orphan_output.go:40 +0x4d
github.com/hashicorp/terraform/terraform.(*OrphanOutputTransformer).Transform(0xc420320470, 0xc4202cfcc0, 0x17, 0x0)
        /opt/gopath/src/github.com/hashicorp/terraform/terraform/transform_orphan_output.go:24 +0x4a
github.com/hashicorp/terraform/terraform.(*BasicGraphBuilder).Build(0xc4206358c0, 0x81d0f50, 0x1, 0x1, 0x0, 0x0, 0x0)
        /opt/gopath/src/github.com/hashicorp/terraform/terraform/graph_builder.go:51 +0x4e2
github.com/hashicorp/terraform/terraform.(*ApplyGraphBuilder).Build(0xc42022c700, 0x81d0f50, 0x1, 0x1, 0x1, 0x554ef5d, 0xc420635960)
        /opt/gopath/src/github.com/hashicorp/terraform/terraform/graph_builder_apply.go:53 +0xc3
github.com/hashicorp/terraform/terraform.(*Context).Graph(0xc4201f4d80, 0xc42013c705, 0xc420635a06, 0x5, 0xc420077ac8, 0x1)
        /opt/gopath/src/github.com/hashicorp/terraform/terraform/context.go:233 +0x94e
github.com/hashicorp/terraform/command.(*GraphCommand).Run(0xc4203adc20, 0xc42000e2a0, 0x2, 0x2, 0xc420400d20)
        /opt/gopath/src/github.com/hashicorp/terraform/command/graph.go:115 +0x62b
github.com/hashicorp/terraform/vendor/github.com/mitchellh/cli.(*CLI).Run(0xc4200ba3c0, 0xc42037ef90, 0xc420635cd0, 0x1)
        /opt/gopath/src/github.com/hashicorp/terraform/vendor/github.com/mitchellh/cli/cli.go:154 +0x1a8
main.wrappedMain(0x0)
        /opt/gopath/src/github.com/hashicorp/terraform/main.go:192 +0xb82
main.realMain(0x0)
        /opt/gopath/src/github.com/hashicorp/terraform/main.go:95 +0xbb
main.main()
        /opt/gopath/src/github.com/hashicorp/terraform/main.go:31 +0x38



!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!

Terraform crashed! This is always indicative of a bug within Terraform.
A crash log has been placed at "crash.log" relative to your current
working directory. It would be immensely helpful if you could please
report the crash with Terraform[1] so that we can fix this.

When reporting bugs, please include your terraform version. That
information is available on the first line of crash.log. You can also
get it by running 'terraform --version' on the command line.

[1]: https://github.com/hashicorp/terraform/issues

!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!

--- a/command/graph.go
+++ b/command/graph.go
@@ -88,6 +88,12 @@ func (c *GraphCommand) Run(args []string) int {
 		return 1
 	}
 
+	// Building a graph may require config module to be present, even if it's
+	// empty.
+	if mod == nil && plan == nil {
+		mod = module.NewEmptyTree()
+	}
+
 	// Build the operation
 	opReq := c.Operation()
 	opReq.Module = mod

--- a/command/graph_test.go
+++ b/command/graph_test.go
@@ -81,6 +81,28 @@ func TestGraph_noArgs(t *testing.T) {
 	}
 }
 
+func TestGraph_noConfig(t *testing.T) {
+	td := tempDir(t)
+	os.MkdirAll(td, 0755)
+	defer os.RemoveAll(td)
+	defer testChdir(t, td)()
+
+	ui := new(cli.MockUi)
+	c := &GraphCommand{
+		Meta: Meta{
+			testingOverrides: metaOverridesForProvider(testProvider()),
+			Ui:               ui,
+		},
+	}
+
+	// Running the graph command without a config should not panic,
+	// but this may be an error at some point in the future.
+	args := []string{"-type", "apply"}
+	if code := c.Run(args); code != 0 {
+		t.Fatalf("bad: \n%s", ui.ErrorWriter.String())
+	}
+}
+
 func TestGraph_plan(t *testing.T) {
 	tmp, cwd := testCwd(t)
 	defer testFixCwd(t, tmp, cwd)

